src/api/errors/api_error.rs

```diff
@@ -24,9 +24,9 @@ pub(in crate::api) enum ApiErrorKind {
     OrganizationNotFound,
     #[error("resource not found")]
     ResourceNotFound,
-    #[error("Project not found. Please check that you entered the project and organization IDs or slugs correctly.")]
+    #[error("Project not found. Please verify that you configured the correct project and organization.")]
     ProjectNotFound,
-    #[error("release not found")]
+    #[error("Release not found. Please verify that you configured the correct release, project, and organization.")]
```

Comment:
I know we're being nice but I recently found that "Please" doesn't fit Sentry's copy very much 😂 I think we could drop it but I'll leave it up to you.